lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1131,6 +1131,30 @@ class ConvertAtenBmmOp : public OpConversionPattern<AtenBmmOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenDropoutOp : public OpConversionPattern<AtenDropoutOp> {
+public:
+  using OpConversionPattern::OpConversionPattern;
+  LogicalResult
+  matchAndRewrite(AtenDropoutOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    if (failed(verifyLinalgCompatibleTypes(op, rewriter)))
+      return failure();
+
+    Location loc = op->getLoc();
```

Comment:
This can be removed after `auto sizes = getTensorSizes(rewriter, loc, input);` line is removed.